### ISSUE TITLE
VANotify edge case dashboard - Design QA

### ISSUE THREAD

**Issue by chloedotbrown** [department-of-veterans-affairs/va.gov-team#38098]
## Issue Description
As we roll out an VANotify edge case dashboard, we need to make sure that the dashboard is easy to use and understand.

---
## Resources
- [VANotify dashboard draft](https://va-gov.domo.com/page/2040841289) 
- Past example tickets – recommend reading through comments:
  - [Design QA for Forms KPIs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24202)
  - [Design QA for Search KPIs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25870)
  - [Design QA for Content KPIs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25871)
  - [Design QA for All-VA Dashboard](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33937)

## Tasks
Use the `Participant Tester` account in Domo to QA for the following:
- [ ] All links should be clickable and correct
- [ ] All tooltips are consistently formatted and accurately describe their metric
- [ ] Directionality of change arrow/colors is correct for the metric
- [ ] All titles and text fully visible in the dashboard layout
- [ ] Filters should work properly
- [ ] Any feedback is documented

## Acceptance Criteria
- [ ] The VANotify dashboard in Domo is usable and follows the design as closely as possible.
---

**Comment by chloedotbrown** [comment]
## Usability checklist

Complete using Participant Tester account – please copy and paste into your own comments below to track QA progress. See example table below or refer to past QA tickets linked above.


| Test                                 | Expected Behavior                                            | Actual Behavior                                              | Pass/Fail |
| ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | --------- |
| Dashboard title                         | Text fully readable, without errors | Dashboard does not have title | Fail ❌  |
| Date filter                         | Every chart gets filtered for date and group by settings are changed |  | Pass ✅  |
| Section titles                       | Every section title & subtitle are spelled correctly          | Sentence case for Landing page.          |Fail ❌    |
| Section resources                       | Links to How-to Guide + Data Dictionary (if available)          |Only How-to needed, no dictionary created           |Fail ❌      |
| Chart titles                          | Every chart title is fully readable, without errors, relevant to chart        |        |Pass ✅      |
| Chart descriptions      | All chart descriptions are disabled, so nothing appears when hovering over title        |    |Pass ✅      |
| Hover text              | All hover text "makes sense" and is spelled correctly        |**_Cards:_** Total notifications sent (use sentence case), Email opens by user (main value), Top reasons not retrieved (include category), Avg. campaign opens per user (main value),    |Fail ❌     |
| Annotations                          | Every annotation "makes sense," is free of errors, and visible within text box      | Can remove Delivery rate by campaign since so similar to note in Highlights. Add annotation below Avg. campaign opens per user. Change "count of unique sessions" to "total sessions" for both text boxes in Landing page section.       |Fail ❌       |
| Filter cards                          | Filter cards are easy to read and use.      | Both filter by campaign cards need more horizontal space so that text is visible. May also need to change color to darker blue, given size of text.      |Fail ❌       |
| Text box formatting    | Every annotation and section title box shows all text, does not generate scroll bar  |    |Pass ✅      |
| Card interactions – filters             | Clicking data point does not filter other cards        | All clickable cards add filters to dashboard  |Fail ❌       |
| Card interactions – card detail             | Clicking chart title opens card in new tab        |All cards open in same tab  |Fail ❌       |
| Display settings – multi-value gauges            | Date grouping ("by week/month") appears under each title     |  | Pass ✅     |
| Display settings – summary numbers            | All non-multivalue gauge charts without a date axis show date range in summary number     | **_Cards:_** Notification delivery rate, Delivery rate by campaign  | Fail ❌    |
| Related cards            | In card detail view, all related card previews have been deleted     |  | Pass ✅       |
| Appendix            | No charts or notecards appear in appendix below dashboard     |  | Pass ✅      |


### Other notes
Outside the scope of the QA template:
- Layout in `Highlights` and `Notifications sent` sections feels a little unbalanced and hard to follow. May want to play around with different options so flow is clearer.
- Color of line charts is still using the Domo default-blue for Total notifications sent and Total users opened emails. Should be changed to one of the blues in the Analytics palette.

CC: @pavanhothi @jonathan-epstein13 @michelle-dooley

**Comment by pavanhothi** [comment]
@chloedotbrown the necessary changes/updates have been made as mentioned in the above QA. Please let me know if any more changes may be needed. Thank you!

**Comment by chloedotbrown** [comment]
## Usability checklist

I made a couple small changes directly to expedite the process:
- Hover text for "Email opens by user" and "Avg. campaign opens by user." Coming up with this text isn't an exact science, but they should be a little longer and provide a bit more info than the card titles alone.
- Added date range to summary number for "Top reasons emails not retrieved." Reordered hover text for clarity (category always comes first, then number).

That said, this should be good to go! Giving this dashboard a clean bill of design-health and closing out the ticket. Thanks for the quick turnaround, @pavanhothi!

| Test                                 | Expected Behavior                                            | Actual Behavior                                              | Pass/Fail |
| ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | --------- |
| Dashboard title       | Text fully readable, without errors |  | Pass ✅ |
| Date filter      | Every chart gets filtered for date and group by settings are changed |  | Pass ✅  |
| Section titles       | Every section title & subtitle are spelled correctly       |    |Pass ✅     |
| Section resources     | Links to How-to Guide + Data Dictionary (if available)          | |Pass ✅      |
| Chart titles                          | Every chart title is fully readable, without errors, relevant to chart        |        |Pass ✅      |
| Chart descriptions      | All chart descriptions are disabled, so nothing appears when hovering over title     |    |Pass ✅      |
| Hover text              | All hover text "makes sense" and is spelled correctly        |   |Pass ✅     |
| Annotations     | Every annotation "makes sense," is free of errors, and visible within text box      |     |Pass ✅    |
| Filter cards        | Filter cards are easy to read and use.      |    |Pass ✅  |
| Text box formatting    | Every annotation and section title box shows all text, does not generate scroll bar  |    |Pass ✅      |
| Card interactions – filters             | Clicking data point does not filter other cards        |   | Pass ✅    |
| Card interactions – card detail             | chart title opens card in new tab        |  |Pass ✅        |
| Display settings – date units            | Date grouping ("by week/month") appears under each title     |  | Pass ✅     |
| Display settings – multi-value gauges            | Directionality of colors in period-over-period comparison makes sense|  | Pass ✅  |
| Display settings – summary numbers            | All non-multivalue gauge charts without a date axis show date range in summary number     |   | Pass ✅     |
| Related cards            | In card detail view, all related card previews have been deleted     |  | Pass ✅       |
| Appendix            | No charts or notecards appear in appendix below dashboard     |  | Pass ✅      |
